app/views/support_interface/application_forms/application_choices/change_offered_course/confirm_offered_course_option.html.erb

```diff
@@ -39,6 +39,13 @@
         <%= f.govuk_check_box :accept_guidance, true, multiple: false, label: { text: 'I have read the guidance' }, link_errors: true %>
       <% end %>
 
+      <% if flash["warning"] == I18n.t('support_interface.errors.messages.course_full_error') %>
```

Comment:
Can we use another way to verify the course full than a comparison with flash against a locale? Like you rescue the Course Full in the controller and add to a variable or check against the course op object that the user want to save?

Reply:
agreed, the controller could assign something along the lines of `@show_course_change_confirmation`

Reply:
Thanks, fixed in f2b6b2